### PR TITLE
HW374: Fixed sticky header height

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
Note:
This fixed dynamic height issues when filters are selected

### CODE REVIEW

**Comment by mukeshcompucorp** [comment]
@colemanw Please review last 2 commits from this

**Comment by colemanw** [comment]
@mukeshcompucorp please remember not to create merge commits in a PR as they make things messy. When updating use the `--rebase` flag e.g. `git pull --rebase origin master` and then use the `-f` flag when pushing changes up to the PR e.g. `git push -f mukeshcompucorp HW-374-sticky-filters-on-feed`

File: ang/civicase/ActivityFilters.js [hunk 1]
```diff
@@ -19,14 +19,21 @@
 
         var $actHeader = $('.act-feed-panel .panel-header'),
         $actControls = $('.act-feed-panel .act-list-controls'),
-        $civicrmMenu = $('#civicrm-menu');
+        $civicrmMenu = $('#civicrm-menu'),
+        $feedActivity = $('.act-feed-view-activity');
```
Comment (colemanw) [inline ang/civicase/ActivityFilters.js]: By convention we indent these 2 spaces, like

    var $actHeader = $('.act-feed-panel .panel-header'),
      $civicrmMenu = $('#civicrm-menu'),
      $feedActivity = $('.act-feed-view-activity');

File: ang/civicase/ActivityFilters.js [hunk 2]
```diff
@@ -19,14 +19,21 @@
...
         
         $('.act-feed-view-activity').affix({
```
Comment (colemanw) [inline ang/civicase/ActivityFilters.js]: You just created a variable for this 2 lines above, why not use it?

File: ang/civicase/ActivityFilters.js [hunk 3]
```diff
@@ -47,6 +54,24 @@
           $actControls.css('width','auto');
           $actControls.css('top', 'auto');
         });
+
+        $scope.$watchCollection('filters', function(){
+          $timeout(function() {
+            $actControls.css('top',$civicrmMenu.height() + $actHeader.height());
+            $feedActivity
+            .height($(window).height() - ($civicrmMenu.height() + $actHeader.height() + $actControls.height()))
+            .css('top',$civicrmMenu.height() + $actHeader.height() + $actControls.height());
```
Comment (colemanw) [inline ang/civicase/ActivityFilters.js]: When chaining methods we indent 2 extra spaces.

File: ang/civicase/ActivityFilters.js [hunk 4]
```diff
@@ -47,6 +54,24 @@
...
+          });
+        });
+
+        $scope.$watchCollection('exposedFilters', function(){
+          $timeout(function() {
+            $actControls.css('top',$civicrmMenu.height() + $actHeader.height());
+            $('#bootstrap-theme .act-feed-view-activity')
+            .height($(window).height() - ($civicrmMenu.height() + $actHeader.height() + $actControls.height()))
+            .css('top',$civicrmMenu.height() + $actHeader.height() + $actControls.height());
```
Comment (colemanw) [inline ang/civicase/ActivityFilters.js]: Ditto

**Pull request closed without merging**